src/DocChecks.jl

```diff
@@ -343,8 +353,8 @@ function report(result::Result, str, doc::Documents.Document)
     println(ioc)
     printstyled(ioc, "> File: ", result.file, "\n", color=:cyan)
     printstyled(ioc, "\n> Code block:\n", color=:cyan)
-    println(ioc, "\n```jldoctest")
-    println(ioc, result.code)
+    println(ioc, "\n```$(result.block.language)")
```

Comment:
Storing the full `Markdown.Code` object also allows to print the actual `language` field here, including `name`, `filter` and `setup`. Previously it was a bit confusing that those things were left out from the error printing.